### ISSUE TITLE
Stripe integration for payment gateway

### ISSUE THREAD

**Issue by arpittiwari24** [arpittiwari24/YourNextSaas.online#2]
Currently the boilerplate supports lemon squeezy for accepting payments . The stripe integration should be there too as a separate route for the user to choose between the two .

**Comment by chaheti89** [comment]
Can I work on this issue?

**Comment by arpittiwari24** [comment]
Yup sure . I am assigning this to you . Can you please take a moment and star the repo  :D
